nrf_modem/doc/sockets.rst

```diff
@@ -353,10 +353,10 @@ Packet data networks
 
 The Modem library supports selecting which Packet Data Network (PDN) to use on a network socket and for DNS queries.
 The configuration of Packet Data Protocol (PDP) contexts, and the activation of PDN connections are not handled by the Modem library.
-To configure PDP contexts and activate PDN connections, the application must use the nRF9160 modem packet domain AT commands.
+To configure PDP contexts and activate PDN connections, the application must use the nRF91 Series modem packet domain AT commands.
 
 When performing network operations on any PDN, the application ensures that the PDN connection is available.
-For more information about how to configure PDP contexts, activate PDN connections, and determine their state, see the `nRF9160 modem Packet Domain AT commands`_ documentation.
+For more information about how to configure PDP contexts, activate PDN connections, and determine their state, see the `nRF91 Series modem Packet Domain AT commands`_ documentation.
```

Comment:
There will also be separate AT command manuals.

Reply:
Agree. I will be fixing this in the commit that I am working for the PR.